package.json

```diff
@@ -12,12 +12,16 @@
     "postcss-cli": "6.1.3",
     "stylelint": "10.1.0",
     "stylelint-config-standard": "18.3.0",
-    "updates": "8.5.3"
+    "updates": "8.5.3",
+    "webpack-cli": "3.3.10"
   },
   "browserslist": [
     "> 1%",
     "last 2 firefox versions",
     "last 2 safari versions",
     "ie 11"
-  ]
+  ],
+  "dependencies": {
+    "webpack": "4.41.2"
+  }
```

Comment:
I'd keep wepack in `devDependencies`. While practically it does not make much difference we can keep a separation between frontend libs and devtools that way.